x509/src/Cardano/X509/Configuration.hs

```diff
@@ -76,6 +76,9 @@ data TLSConfiguration = TLSConfiguration
 instance FromJSON TLSConfiguration where
     parseJSON = Aeson.genericParseJSON (aesonDropPrefix "tls")
 
+instance ToJSON TLSConfiguration where
```

Comment:
These instances should really be made explicit. This is the approach the core team took because (as we discovered) it's a lot more difficult to debug JSON serialization issues when the instances aren't explicit.